src/api/db/seeds.rb

```diff
@@ -54,6 +54,8 @@
 admin = User.where(login: 'Admin').first_or_create(login: 'Admin', email: 'root@localhost',
                                                     realname: 'OBS Instance Superuser', state: 'confirmed',
                                                     password: 'opensuse')
+admin.is_beta = true && admin.save! if Rails.env.development?
```

Comment:
`admin.update!(in_beta: true) if Rails.env.development?`

Reply:
There is also a typo, it is `in_beta`, not `is_beta`

Reply:
Thanks, fixed!